packages/santa/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.2.0"
+  changes:
+    - description: Update log format
```

How about something like this to give a bit more detail...

```suggestion
    - description: Update log format to support the GA releases of Santa. The pre-GA Santa log format (circa 2017) is no longer accepted.
```